src/tools/base/BaseBrushTool.js

```diff
@@ -18,9 +16,11 @@ const { state, setters } = store.modules.brush;
 class BaseBrushTool extends BaseTool {
   constructor(props, defaultProps = {}) {
     if (!defaultProps.configuration) {
-      defaultProps.configuration = {};
+      defaultProps.configuration = { alwaysEraseOnClick: false };
     }
     defaultProps.configuration.referencedToolData = 'brush';
+    defaultProps.mixins = ['segmentationAPI'];
```

Comment:
👀 Virtual instead of default mixin? They would be overriden by an mixins added, or you can modify the prototype?

Reply:
Yeah, looking at this, this straight up breaks things... we will get rid of the `segmentationAPI` anyways 👍 .